src/Transports.AspNetCore/WebSockets/WebSocketHandler.cs

```diff
@@ -0,0 +1,165 @@
+using System.Net.WebSockets;
+using GraphQL.Transport;
+using GraphQL.Types;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets;
+
+/// <inheritdoc cref="WebSocketHandler"/>
+public class WebSocketHandler<TSchema> : WebSocketHandler, IWebSocketHandler<TSchema>
+    where TSchema : ISchema
+{
+    /// <inheritdoc cref="WebSocketHandler(IGraphQLSerializer, IDocumentExecuter, IServiceScopeFactory, GraphQLWebSocketOptions, IAuthorizationOptions, IHostApplicationLifetime, IWebSocketAuthenticationService)"/>
+    public WebSocketHandler(
+        IGraphQLSerializer serializer,
+        IDocumentExecuter<TSchema> executer,
+        IServiceScopeFactory serviceScopeFactory,
+        GraphQLWebSocketOptions options,
+        IAuthorizationOptions authorizationOptions,
+        IHostApplicationLifetime hostApplicationLifetime,
+        IWebSocketAuthenticationService? authorizationService)
+        : base(serializer, executer, serviceScopeFactory, options, authorizationOptions, hostApplicationLifetime, authorizationService)
+    {
+    }
+
+    /// <inheritdoc cref="WebSocketHandler(IGraphQLSerializer, IDocumentExecuter, IServiceScopeFactory, GraphQLWebSocketOptions, IAuthorizationOptions, IHostApplicationLifetime, IWebSocketAuthenticationService)"/>
```

Comment:
no IWebSocketAuthenticationService here

Reply:
That is correct.  All intellisense comments are accurate, however.